### PR TITLE
Add ipaddr.js w/ git auto-update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #9909

@cdnjs/intern3 Please help me review this PR. Thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/whitequark/ipaddr.js
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/ipaddr.js
 * License and its reference: MIT, [ref](https://github.com/whitequark/ipaddr.js/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 11
   - Count of stars: 166
   - Count of forks: 26
 * NPM download stats (optional):
   - Downloads in the last day: 377,207
   - Downloads in the last week: 1,836,029
   - Downloads in the last month: 7,504,477

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Ping @extend1994

**Comment by hare1039** [comment]
@pvnr0082t I've rebased to master branch. Please help me review this PR. Thanks.

File: ajax/libs/ipaddr.js/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/whitequark/ipaddr.js.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "ipaddr.min.js"
```
Comment (pvnr0082t) [inline ajax/libs/ipaddr.js/package.json]: Let's use `"ipaddr*"`.

File: ajax/libs/ipaddr.js/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        ]
+      },
+      {
+        "basePath": "lib",
+        "files": [
+          "ipaddr.js"
```
Comment (pvnr0082t) [inline ajax/libs/ipaddr.js/package.json]: Let's use `"ipaddr*"`.

**Comment by hare1039** [comment]
@pvnr0082t I added version `1.1.1` and followed the better rule in package.json. Please help me reviewing this PR. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t I've amend the commit message more easier to read. Please review this PR again. Thanks.

**Comment by PeterDaveHello** [comment]
There should be a noun following `get`

**Comment by hare1039** [comment]
@pvnr0082t I've amend the commit message more easier to read. Please review this PR again. Thanks.

**Comment by PeterDaveHello** [comment]
The first commit message you use `ipaddr.js` but the second `ipaddr`? As we said before, the second commit you don't need to mention "old", just its version.

**Comment by hare1039** [comment]
@pvnr0082t I've amend the commit message more easier to read. Please review this PR again. Thanks.

**Comment by whitequark** [comment]
I get an email every time this PR is updated...

**Comment by PeterDaveHello** [comment]
@whitequark sorry about that, if this is disturbing you, I'll respectfully suggest you to click 'unsubscribe' button at the right side, we'll get this merged as soon as possible, thank you.

**Comment by whitequark** [comment]
I can't. The mail comes because I'm mentioned in a commit that's being amended.

**Comment by PeterDaveHello** [comment]
@whitequark sorry about that, we'll get this merged as soon as possible, promise.

**Comment by PeterDaveHello** [comment]
Please update the commit to remove the author, I don't want to disturb the author any more.
There are still problems in the second commit message.
1. The first line, `Manually add ipaddr.js v1.1.1, cc #9909` is better
2. `Manually add v1.1.1 due to git auto-update can't get it from npm` -> of course git can't fetch update from npm ... this description is odd, please reference to the other familiar commits.

**Comment by hare1039** [comment]
@pvnr0082t I've amend the commit message more easier to read. Please review this PR again. Thanks.

**Pull request merged**